server/src/bot.js

```diff
@@ -2,32 +2,23 @@ import Botkit from 'botkit';
 import dotenv from 'dotenv';
```

Comment:
Do we need this anymore? I think .env works without it?? Not sure, pls confirm w/ rest of team